ci/scripts/docker-hdfs.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Exits as soon as any line fails.
+set -euo pipefail
+
+ghcraddr="ghcr.io/risingwavelabs/risingwave"
+dockerhubaddr="risingwavelabs/risingwave"
+arch="$(uname -m)"
+
+java_home_path=$(uname -m)
+if [ "$arch" = "arm64" ] || [ "$arch" = "aarch64" ]; then
+    java_home_path="/usr/lib/jvm/java-11-openjdk-arm64"
+else 
+# x86_64
+    java_home_path="/usr/lib/jvm/java-11-openjdk-amd64"
+fi
+echo $java_home_path
+
+# Build RisingWave docker image ${BUILDKITE_COMMIT}-${arch}
+echo "--- docker build and tag"
+docker build -f docker/Dockerfile --build-arg "GIT_SHA=${BUILDKITE_COMMIT}" --build-arg "JAVA_HOME_PATH=${java_home_path}" -t "${ghcraddr}:${BUILDKITE_COMMIT}-${arch}" --target risingwave .
```

Comment:
If this script is only used for manual execution, please remove all buildkite parameters, otherwise the script will not succeed. 
like `BUILDKITE_COMMIT `

Reply:
Thanks for pointing out. I replace it with an auto-generated random number. `BUILDKITE_COMMIT="HDFS_$(echo $RANDOM | md5sum | head -c 20;)"` Does it looks good to you?